Improvement: Adds a chat filter to being hit by a venom shot

BearySuperior opened the pull request:
## What
https://discord.com/channels/997079228510117908/1295179982078742590/1295179982078742590
Adds a chat filter to being hit by a venom shot

## Changelog Improvements
+ Added Venom Shot attacks to the Arachne chat filter. - BearySuperior

BearySuperior committed:
Adds a chat filter to being hit by a venom shot

CalMWolfs commented on src/main/java/at/hannibal2/skyhanni/features/chat/VenomShotHider.kt:
```diff
@@ -0,0 +1,33 @@
+
+    private val config get() = SkyHanniMod.feature.chat
+
+    private val VenomShotPattern = Regex(".*Venom Shot.*", RegexOption.IGNORE_CASE)
```
Should be a repo pattern, see usages in the rest of the mod for more info

CalMWolfs reviewed (changes requested):
Will probably have more requested changes once this is changed

BearySuperior committed:
## What
https://discord.com/channels/997079228510117908/1295179982078742590/1315162402828058654
Adds a chat filter for Arachne's venom shot

## Changelog New Features
+ Adds a chat filter for the Venom Shot - BearySuperior

BearySuperior committed:
## What
https://discord.com/channels/997079228510117908/1295179982078742590/1315162402828058654
Adds a chat filter for Arachne's venom shot

## Changelog New Features
+ Adds a chat filter for the Venom Shot - BearySuperior

BearySuperior committed:
## What
https://discord.com/channels/997079228510117908/1295179982078742590/1315162402828058654
Adds a chat filter for Arachne's venom shot

## Changelog New Features
+ Adds a chat filter for the Venom Shot - BearySuperior

BearySuperior committed:
Fixed formatting problems

CalMWolfs committed:
fix errors

CalMWolfs committed:
Merge branch 'beta' into Feature-Venom-Shot-Chat-Hider

hannibal002 committed:
Merge branch 'refs/heads/beta' into fork/BearySuperior/Feature-Venom-Shot-Chat-Hider

hannibal002 committed:
formatting

CalMWolfs committed:
testings

CalMWolfs committed:
baseline
